include/flatcc/portable/pdiagnostic.h

```diff
@@ -20,12 +20,22 @@
 
 #if defined(_MSC_VER) && !defined(PDIAGNOSTIC_AWARE_MSVC)
 #define PDIAGNOSTIC_AWARE_MSVC 1
-#elif defined(__clang__) && !defined(PDIAGNOSTIC_AWARE_CLANG)
+#elif !defined(PDIAGNOSTIC_AWARE_MSVC)
+#define PDIAGNOSTIC_AWARE_MSVC 0
+#endif
+
+#if defined(__clang__) && !defined(PDIAGNOSTIC_AWARE_CLANG)
```

Comment:
Here you have the feature flag last unlike in paligned_alloc - I know this follows precedence in the current file, but we should probably choose one consistent approach.

Reply:
I have no preference; if it's styled differently elsewhere I can swap these around.